json-schemas/interface-methods/protocol-rule-set.json

```diff
@@ -66,6 +66,7 @@
                 "enum": [
                   "delete",
                   "query",
+                  "subscribe",
```

Comment:
Needing clarification on the requirements of this feature base on the statement "follows similar rules as those for `RecordsQuery`":

Without a protocol definition:
1. Can anonymous (unauthenticated) people subscribe to record streams?
1. Can authenticated recipient or author subscribe to record streams?

Reply:
Yeah I think I need to describe this a bit better somewhere. I will add a section to the readme that describes subscriptions in general and these rules.

To answer your questions:
1. Yes, they will receive messages that are marked `published:true`.

Not sure if I fully understand the question in 2. If you'd like to chat about that we can.

Reply:
For #2, I was wondering about if the subscription is signed, but does not reference a permission or role, or protocol. Basically was imagining in my head what the perf would be like. I guess I will learn more as I read on.

Reply:
If I understand correctly this is the flow you mean:
https://github.com/TBD54566975/dwn-sdk-js/blob/dff724aacd457f695de6f70655b787f7ab6fc073/tests/scenarios/subscriptions.spec.ts#L972

Which would filter based on the author of the subscription being the author or recipient of a record. However this runs into the edge case issue which you found in https://github.com/TBD54566975/dwn-sdk-js/issues/669